### PR TITLE
modify description of static vs dynamic sites

### PR DESCRIPTION
**Pull request by aleathwick**
I would like to suggest another angle for introducing static vs dynamic sites. I think that as it currently stands it could be somewhat confusing to newcomers, e.g. 'updated after loading' - does 'updated' refer to something changing server side only ready for the next refresh, or is this referring to the page being updated in browser in response to further communications between the client and server? 'information continuuing to be passed between browser and server' - what about dynamic sites which are generated then sent once, with no further communication between browser and server until the next refresh?

At the risk of making it more wordy, I've modified the paragraph with a different take on it.

### CODE REVIEW

File: _episodes/introduction.md [hunk 1]
```diff
@@ -143,15 +143,19 @@ individual **l**ist **i**tem (`<li>` and `</li>`) tags.
...
-with information continuing to be passed between browser and server,
-are called _dynamic_ (for example, pages showing current date and time or shopping baskets, Twitter or Facebook pages,
-which might be updated between each two page refreshes).
-This lesson focuses on static sites and tools that can be used to create them,
-known as **Static Site Generators**.
+_Static_ pages are those whose contents are stored on a server in a state ready
+to be sent to any user who makes a request for that web page. When a request is
+made, the server only needs to send the information comprising that web page
+(such as HTML and CSS). Sites that do not change often, such as a website
+containing one’s CV, are often stored as static sites. Conversely, _dynamic_
```
Comment (tobyhodges) [inline _episodes/introduction.md]:
```suggestion
containing one’s CV, are often stored as static sites. 

Conversely, _dynamic_
```

Suggest to split the paragraph in two at this point.

**Review by tobyhodges — APPROVED** [review]
Thank you @aleathwick I agree that this change improves the clarity of the explanation. I have made one suggestion, to split this into two paragraphs.

We would like to fully credit every contribution to the lesson, so please also add your name to the list in the `AUTHORS` file. You will see that some of us have included our email address there but that is completely optional and there is no obligation for you to do so.

File: _episodes/introduction.md (hunk 1 above)
Comment (aleathwick) [reply _episodes/introduction.md]: Splitting that paragraph looks good to me.

**Comment by tobyhodges** [comment]
Thanks @aleathwick !

**Pull request merged**